helm/node-exporter-app/values.yaml

```diff
@@ -44,7 +44,39 @@ prometheus:
     ##
     selectorOverride: {}
 
-    relabelings: []
+    relabelings:
+    - action: replace
+      separator: ;
+      regex: ";(.*)"
+      replacement: $1
+      sourceLabels:
+      - namespace
+      - __meta_kubernetes_namespace
```

Comment:
I don't get why we have `namespace` as a source label, if its not used.

Reply:
All the relabeling comes written here comes from there : https://github.com/giantswarm/kyverno-policies-observability/blob/3c802cb28356c49e0347adc502e8c8eceefc95cf/helm/kyverno-policies-observability/templates/ServiceMonitor.yaml

Reply:
Alright, otherwise I would have suggested to make this simpler. But there might be a reason for this ... or not.
Fine with me, let's move on.